src/Fmt_ast.ml

```diff
@@ -1603,16 +1603,34 @@ and fmt_expression c ?(box= true) ?epi ?eol ?parens ?ext ({ast= exp} as xexp)
         $ fmt_atrs )
   | Pexp_open (flag, {txt; loc}, e0) ->
       let override = Poly.(flag = Override) in
+      (* open_style = `Long means we use let open Module in (...) style;
+         open_style = `Short means we use Module.(...) style; open_style =
+         `Auto means we use the one fitting best. *)
```

Comment:
My feeling is that comments of this form, which indicate what is done, rather than how the implementation achieves it, ought to instead be incorporated into the doc for the option in Conf.